setup.py

```diff
@@ -17,12 +17,12 @@
 
 
 setup(
-    name='glyphs2ufo',
+    name='glyphsLib',
     version='1.0',
-    packages=['glyphs2ufo'],
+    packages=['glyphsLib'],
     entry_points={
         "console_scripts": [
-            "glyphs2ufo = glyphs2ufo.__main__:main"
+            "glyphsLib = glyphsLib.__main__:main"
```

Comment:
I think `ufo2glyphs` could be a separate script. The name "glyphsLib" sounds more appropriate for the library rather than for it's executable.